healthcheck/healthcheck_test.go

```diff
@@ -5,10 +5,10 @@ import (
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/ahl5esoft/golang-underscore"
```

Comment:
Just looking at the repo, this seems like a fairly underutilized Go library, do we think it's worth introducing the dependency if the package stops being maintained?